demo/index.html

```diff
@@ -1,222 +1,332 @@
+          >
+            firstName + " " + lastName
+          </button>
+          <button
+            class="example-button"
+            onclick="loadExample('name = &quot;John&quot;')"
+          >
+            name = "John"
+          </button>
+          <button
+            class="example-button"
+            onclick='loadExample(&apos;INCLUDES("Red;Blue;Green", "Blue")&apos;)'
+          >
+            INCLUDES("Red;Blue;Green", "Blue")
+          </button>
+          <button
+            class="example-button"
+            onclick="loadExample('INCLUDES(skills, &quot;TypeScript&quot;)')"
+          >
+            INCLUDES(skills, "TypeScript")
+          </button>
+          <button
+            class="example-button"
+            onclick='loadExample(&apos;COUNTMATCHES("pass;passed;passing", "pass")&apos;)'
+          >
+            COUNTMATCHES('pass;passed;passing', "pass")
+          </button>
 
-                <h4>List Operations:</h4>
-                <button class="example-button" onclick="loadExample('INCLUDES(items, &quot;Passed&quot;)')">INCLUDES(items, "Passed")</button>
-                <button class="example-button" onclick="loadExample('COUNTMATCHES(items, &quot;Passed&quot;)')">COUNTMATCHES(items, "Passed")</button>
-                <button class="example-button" onclick="loadExample('LEN(items)')">LEN(items)</button>
-                <button class="example-button" onclick="loadExample('SUM(issues)')">SUM(issues)</button>
-                <button class="example-button" onclick="loadExample('REMOVEBLANKS([1,undefined,2,3])')">REMOVEBLANKS([1,undefined,2,"",3])</button>
-                <button class="example-button" onclick="loadExample('JOIN(items, &quot;;&quot;)')">JOIN([items], ";")</button>
+          <h4>List Operations:</h4>
+          <button
+            class="example-button"
+            onclick="loadExample('INCLUDES(items, &quot;Passed&quot;)')"
+          >
+            INCLUDES(items, "Passed")
+          </button>
+          <button
+            class="example-button"
+            onclick="loadExample('COUNTMATCHES(items, &quot;Passed&quot;)')"
+          >
+            COUNTMATCHES(items, "Passed")
+          </button>
+          <button class="example-button" onclick="loadExample('LEN(items)')">
+            LEN(items)
+          </button>
+          <button class="example-button" onclick="loadExample('SUM(issues)')">
+            SUM(issues)
+          </button>
+          <button
+            class="example-button"
+            onclick="loadExample('REMOVEBLANKS([1,undefined,2,3])')"
+          >
+            REMOVEBLANKS([1,undefined,2,"",3])
+          </button>
+          <button
+            class="example-button"
+            onclick="loadExample('JOIN(items, &quot;;&quot;)')"
+          >
+            JOIN([items], ";")
+          </button>
 
+          <h4>Date Operations:</h4>
+          <button
+            class="example-button"
+            onclick="loadExample('DATEVALUE(dateString)')"
```

Comment:
dateString does not exist?
```suggestion
            onclick="loadExample('DATEVALUE(date1')"
```